ulyp-agent-core/src/main/java/com/ulyp/agent/queue/RecordingQueue.java

```diff
@@ -128,6 +128,9 @@ public void enqueueMethodExit(int recordingId, int callId, Object returnValue, b
     }
 
     private Object[] convert(Object[] args) {
```

Comment:
Optional?

Reply:
Sadly, we can not. Unlike Rust/C++, Java `Optional` is not zero-cost. In Ulyp we strive for maximum performance for our clients' recorded applications even at the cost of readability.